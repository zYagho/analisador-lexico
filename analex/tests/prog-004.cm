int soma (int u, int v){
  return u + v;
}

int main(void){
  int x; int y; int z;
  x = input();
  y = input();
  z = soma(x,y);
  output(z);
  return 0;
}

