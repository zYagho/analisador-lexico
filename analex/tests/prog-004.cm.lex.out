INT
ID
LPAREN
INT
ID
COMMA
INT
ID
RPAREN
LBRACES
RETURN
ID
PLUS
ID
SEMICOLON
RBRACES
INT
ID
LPAREN
VOID
RPAREN
LBRACES
INT
ID
SEMICOLON
INT
ID
SEMICOLON
INT
ID
SEMICOLON
INT
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
ATTRIBUTION
ID
LPAREN
ID
COMMA
ID
RPAREN
SEMICOLON
ID
LPAREN
ID
RPAREN
SEMICOLON
RETURN
NUMBER
SEMICOLON
RBRACES
